crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -876,8 +887,18 @@ mod tests {
         let upnp = prepare_test_setup().await;
         tokio::time::sleep(Duration::from_millis(100 + 20)).await;
 
-        // Upnp will change the static port to a random port
-        assert!(ENDPOINT.lock().wg.port() != 1000);
+        let udp_int_port = ENDPOINT.lock().udp.port();
+        let wg_int_port = ENDPOINT.lock().wg.port();
+
+        assert_eq!(udp_int_port, 55555);
+        assert_eq!(wg_int_port, 55345);
+
+        let epc = upnp.get_endpoint_candidate().await.unwrap();
+        let epc_wg_port = epc.wg.port();
+        let epc_udp_port = epc.udp.port();
+
+        assert!(MIN_EPHEMERAL_PORT_RANGE < epc_wg_port && MAX_EPHEMERAL_PORT_RANGE > epc_wg_port);
+        assert!(MIN_EPHEMERAL_PORT_RANGE < epc_udp_port && MAX_EPHEMERAL_PORT_RANGE > epc_udp_port);
```

Comment:
Isn't low range bound inclusive?

Reply:
changed constqnt to be of inclusive range type.